src/experimental/safe_html_transform/interfaces.py

```diff
@@ -2,6 +2,34 @@
 """Module where all interfaces, events and exceptions live."""
 
 from zope.publisher.interfaces.browser import IDefaultBrowserLayer
+from z3c.form import interfaces
+ 
+from zope import schema
+from zope.interface import Interface
+ 
+from zope.i18nmessageid import MessageFactory
+ 
+_ = MessageFactory('experimental.safe_html_transform')
+ 
+ 
+class IExperimenatalSafe_Html_TransformSettings(Interface):
+"""Global safe_html_transform settings. This describes records stored in the
+	configuration registry and obtainable via plone.registry.
+"""
+ 
+    Experimentalsafe_html_transform_key = schema.TextLine(title=_(u"Experimenatalsafe_html_transform (Wordpress) Key"),
```

Comment:
"Also you should use the schema from the CMFPlone pull request in the
Google docs document." Didn't get that part ?

Prakhar Joshi
DA-IICT,Gandhinagar

On Mon, Feb 9, 2015 at 10:18 PM, Timo Stollenwerk <notifications@github.com>
wrote:

> In src/experimental/safe_html_transform/interfaces.py
> <https://github.com/collective/experimental.safe_html_transform/pull/1#discussion_r24342905>
> :
>
> > +from z3c.form import interfaces
> > +
> > +from zope import schema
> > +from zope.interface import Interface
> > +
> > +from zope.i18nmessageid import MessageFactory
> > +
> > +_ = MessageFactory('experimental.safe_html_transform')
> > +
> > +
> > +class IExperimenatalSafe_Html_TransformSettings(Interface):
> > +"""Global safe_html_transform settings. This describes records stored in the
> > +	configuration registry and obtainable via plone.registry.
> > +"""
> > +
> > +    Experimentalsafe_html_transform_key = schema.TextLine(title=_(u"Experimenatalsafe_html_transform (Wordpress) Key"),
>
> There is no need to use an "Experimentalsafe_html" prefix for everything.
> Also you should use the schema from the CMFPlone pull request in the Google
> docs document.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/collective/experimental.safe_html_transform/pull/1/files#r24342905>
> .
>